README.md

```diff
@@ -214,3 +214,5 @@ Note that this change might cause problems with other Java-based applications (e
 
 #### gradlew Failure: `java.lang.NullPointerException (no error message).`
 If you encounter the `java.lang.NullPointerException (no error message).` when running `gradlew`, please make sure your Java version for this project is Java 8.
+
+This can be configured under **File > Project Structure** in Android Studio, or by editing `~/.gradle/gradle.properties` to set `org.gradle.java.home=/path/to/jdk` for the command line.
```

Comment:
@yanokwa I vaguely remember that when this issue was added you didn't want to add a solution that was too platform-specific. The `~/` and forward slashes kind of imply a *NIX environment but I think it's generic enough to be understandable to a Windows user. What do you think?

Reply:
Agreed that we'd like to be platform agnostic. 

`$USER_HOME/.gradle/gradle.properties` is what the Gradle docs use. Se

Reply:
🤷🏻‍♀️ Not a big difference but always good to match official docs.

Reply:
Agreed, will update :+1: